provider/src/commonMain/kotlin/at/asitplus/crypto/provider/os/Attestation.kt

```diff
@@ -0,0 +1,63 @@
+package at.asitplus.crypto.provider.os
```

Comment:
should this be in datatypes somewhere? open to input

Reply:
probably